lib/InvoiceItem.php

```diff
@@ -16,13 +16,14 @@
  * @property string|null $invoice
  * @property bool $livemode
  * @property \Stripe\StripeObject $metadata
- * @property mixed $period
+ * @property \Stripe\StripeObject $period
  * @property \Stripe\Plan|null $plan
  * @property bool $proration
  * @property int $quantity
  * @property string|null $subscription
  * @property string $subscription_item
- * @property array|null $tax_rates
+ * @property \Stripe\TaxRate[]|null $tax_rates
+ * @property bool $unified_proration
```

Comment:
this is interesting

Reply:
Yeah that one is problematic because it was ungated by mistake, so it stays in the spec to avoid a major version.
In a perfect world, we'd blacklist this in php at the override level